Assets/Scripts/Player/PlayerController.cs

```diff
@@ -79,7 +88,6 @@ void Awake()
         //slash.canceled += ctx => cameraLens.FieldOfView = defaultFOV;
        // slash.canceled += ctx => StartZoom(false);
     }
-
     /// <summary>
```

Comment:
Clean up unused code.

Reply:
sorry it was import code from another game from nick g I removed now